### PR TITLE
Add keymap descriptions for Ruby module

### PR DESCRIPTION
**Pull request by lazyatom**
We can make the whichkey menu for Ruby files a bit more user friendly by showing simple names for each section.

I had hoped to set the description for the projectile commands too, but I can't figure out how to set it up since it doesn't actually use a "prefix" in the normal way. If anyone knows how, I'll gladly implement it 😄

### CODE REVIEW

File: modules/lang/ruby/config.el [hunk 1]
```diff
@@ -136,7 +136,7 @@
   :config
   (setq rspec-use-rvm (executable-find "rvm"))
   (map! :localleader
-        :prefix "t"
+        :prefix ("t" . "test")
```
Comment (hlissner) [inline modules/lang/ruby/config.el:139]: Which-key labels aren't supported on minor mode keymaps. This is actually create a global label for all `SPC m t` keybinds. Please remove this.

File: modules/lang/ruby/config.el [hunk 2]
```diff
@@ -164,7 +164,7 @@
     (add-hook 'minitest-mode-hook #'evil-normalize-keymaps))
   (map! :localleader
         :map minitest-mode-map
-        :prefix "t"
+        :prefix ("t" . "test")
```
Comment (hlissner) [inline modules/lang/ruby/config.el:167]: Which-key labels aren't supported on minor mode keymaps. This is actually create a global label for all `SPC m t` keybinds. Please remove this.

File: modules/lang/ruby/config.el (hunk 1 above)
Comment (lazyatom) [reply modules/lang/ruby/config.el]: Removed in the latest push, apologies, I didn't realise.

**Comment by hlissner** [comment]
Thanks!

**Pull request merged**